.github/workflows/main.yml

```diff
@@ -56,6 +56,23 @@ jobs:
       - name: Test public API
         run: ./gradlew :apiTester:assemble
 
+  test-unit-tests:
```

Comment:
Sounds good! When you migrate it to CircleCI later, don't forget that `./gradlew test` only runs the JVM tests and we'll need to figure something else out for the iOS tests 😅

Reply:
Yeah on that topic, just realized today that `./gradlew allTests` should also run the iOS tests 😄

Reply:
Oh nice! I changed the CI additions in this PR to run `./gradlew allTests` instead of `./gradlew test` here: https://github.com/RevenueCat/purchases-kmp/pull/139/commits/f83eaac0f5ced3d675f674d252eebfb19f4b752a